expression/builtin_time.go

```diff
@@ -357,10 +357,10 @@ func (b *builtinDateLiteralSig) Clone() builtinFunc {
 func (b *builtinDateLiteralSig) evalTime(row chunk.Row) (types.Time, bool, error) {
```

Comment:
The error information in vecEvalTime (vectorized implementation) also need to be update.

Reply:
Done